bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -601,7 +601,8 @@ private IUnknown createControllerInitializationCallback(int previousAttempts) {
 		webViewProvider.abortInitialization();
 		initializationRollback.run();
 	};
-	return newCallback((result, pv) -> {
+	return newCallback((resultAsLong, pv) -> {
+		int result = (int) resultAsLong;
```

Comment:
```suggestion
		int result = (int) resultAsLong & 0xFFFFFF;
```

I think this would be the correct way to truncate an unsigned long to an int.

Reply:
`resultAsLong` is a value passed via JNI that was converted from native unsigned long to jint. I have no idea how to best truncate that to Java int, but as said above: this all should be revised independently. The change in this PR just sticks to existing conversion behavior (for which we know it worked expect for this line that did a wrong comparison).
So would not be in favor of changing conversion here with no real gain and in the worst case the risk of a regression.